src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -345,32 +347,36 @@ export const MoreActionsDrawer = ({
             <ListItem disablePadding={false}>
               <Typography variant="body2">Layout Thoroughness</Typography>
               <Tooltip
-                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. 1 = lowest effort, 100 = highest effort."
+                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. Low = minimal effort, High = maximum effort."
                 enterTouchDelay={0} // allow touch to immediately trigger
                 leaveTouchDelay={Infinity} // touch-away to close on mobile, since message is long
               >
                 <IconButton
                   color="info"
                   aria-label="Thoroughness info"
                   sx={{
-                    // Don't make it look like clicking will do something, since it won't.
-                    // Using a button here is an attempt to make it accessible, since the tooltip will show
-                    // on focus.
-                    cursor: "default",
+                    cursor: "default", // Maintain accessibility while preventing click actions
                     alignSelf: "center",
                   }}
                 >
                   <Info />
                 </IconButton>
               </Tooltip>
-              <NumberInput
-                min={1}
-                max={100}
+              <Select
                 value={layoutThoroughness}
-                onChange={(_event, value) => {
-                  if (value) setLayoutThoroughness(value);
+                onChange={(event) => {
+                  const value = event.target.value;
+                  setLayoutThoroughness(Number(value));
                 }}
```

Comment:
```suggestion
                }}
                size="small"
```

I don't think our box needs as much space as Mui gives it by default, so I'd add the prop `size="small"` here